corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -641,6 +644,25 @@ hqDefine('geospatial/js/models', [
             updateSelectedSavedPolygonParam();
         };
 
+        self.deleteSelectedPolygonFilter = function () {
```

Comment:
Does a confirmation popup appear when the user tries to do a delete? If not, it might be good to add.

Reply:
Agreed. Good to have it.

Reply:
Added in https://github.com/dimagi/commcare-hq/pull/34717/commits/974ef8633067182b8839e2282ff2f92c96832678